python/tests/conftest.py

```diff
@@ -397,7 +400,18 @@ def mongo_store_factory(request, lib_name):
         mongo_server_sess = request.getfixturevalue("mongo_server_sess")
         uri = f"mongodb://{mongo_server_sess.hostname}:{mongo_server_sess.port}"
 
-    cfg_maker = functools.partial(create_test_mongo_cfg, uri=uri)
+    yield uri
+
+    # mongodb does not support prefix to differenciate different tests and the server is session-scoped
+    mongo_client = pymongo.MongoClient(uri)
+    for db_name in mongo_client.list_database_names():
+        if db_name != "local" and db_name != "admin" and db_name != "apiomat" and db_name != "config":
+            mongo_client.drop_database(db_name)
```

Comment:
This is not reliable if the same Mongo server is used to support concurrent tests.

Just use the `clear()` method.

Reply:
Thanks for heads up. I do miss your points. However, I am not sure whether this is a clean solution either. 
The clear up is for handling checking `assert ac.list_libraries() == []`. I believe it is for ensuring the test environment is clear. Yet I don't believe the checking is still valid if we run tests parallelly.
I think checking the particular library used for testing should be enough.

Reply:
Using `clear()` needs to keep track of the libraries in use. It will be a bit tricky to do so for uri related test